README.rst

```diff
@@ -21,8 +21,8 @@ andi
 .. warning::
     Current status is "experimental".
 
-``andi`` tells which kwargs should to be passed to a callable, based
-on callable arguments' type annotations.
+``andi`` makes easy implementing custom dependency injection mechanisms
+where dependencies are expressed using type annotations.
```

Comment:
heh, in a previous version I tried hard to avoid mentioning "dependency injection" unless absolutely required :)

Reply:
hehe. There are other alternatives like explaining it as "recursive inspection mechanism using type annotations" or something like that... But I think it would be even more obscure than "dependency injection" if that is possible :-)